scripts/bootstrap-aligner-api-secret.sh

```
@@ -0,0 +1,212 @@
+# Parse and normalize secret file entries
+declare -A FOUND_KEYS
+tmp_env_dir=$(mktemp -d)
+trap 'rm -rf "$tmp_env_dir"' EXIT
+clean_env_file="$tmp_env_dir/secrets.env"
+
+while IFS= read -r line || [ -n "$line" ]; do
+  trimmed=$(echo "$line" | sed -e 's/^[[:blank:]]*//' -e 's/[[:blank:]]*$//')
+  [ -z "$trimmed" ] && continue
+  [[ "$trimmed" =~ ^# ]] && continue
+  trimmed="${trimmed#export }"
+  trimmed=$(echo "$trimmed" | sed -e 's/^[[:blank:]]*//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)